.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,3 +10,7 @@ Please see our [OSS process document](https://github.com/honeycombio/home/blob/m
 ## Short description of the changes
 
 ## How to verify that this has the expected result
+
+---
+
+- [ ] Changelog is updated
```

Comment:
i've been forgetting to update the changelog in my PRs so figured a checkbox in the PR template would be useful!